x-pack/plugins/ml/public/application/components/data_grid/column_chart.tsx

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC } from 'react';
+import classNames from 'classnames';
+
+import { BarSeries, Chart, Settings } from '@elastic/charts';
+import { EuiDataGridColumn } from '@elastic/eui';
+
+import './column_chart.scss';
+
+import {
+  isNumericChartData,
+  isUnsupportedChartData,
+  useColumnChart,
+  ChartData,
+} from './use_column_chart';
+
+interface Props {
+  chartData: ChartData;
+  columnType: EuiDataGridColumn;
+}
+
+export const ColumnChart: FC<Props> = ({ chartData, columnType }) => {
+  const { data, legendText, xScaleType } = useColumnChart(chartData, columnType);
+
+  return (
+    <>
+      {!isUnsupportedChartData(chartData) && data.length > 0 && (
```

Comment:
Need to think of some text to use in this case where the field contains no data :)

![image](https://user-images.githubusercontent.com/7405507/84913056-fedfc800-b0b1-11ea-97f7-c1d583e0fd2c.png)

Reply:
Fixed in 8570a4b.